app/static/app/js/components/ErrorMessage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class ErrorMessage extends React.Component {
+    constructor(props){
+        super();
+
+        this.state = {
+            error: props.message
+        };
+
+        this.close = this.close.bind(this);
+    }
+
+    close(){
+        this.setState({error: ""});
+    }
+
+    render(){
+        if (this.state.error !== ""){
+            return (
+                <div className="alert alert-warning alert-dismissible">
```

Comment:
I was having good success with [React Bootstrap](https://react-bootstrap.github.io/) last week. It helped clear up the handling of modals particularly.